From 1e853b937b2fcc51ff3939c7ae657d0726681ca1 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Wed, 17 Mar 2021 09:36:58 +0000
Subject: [PATCH] cifsd: Fix a handful of spelling mistakes
Git-commit: 1e853b937b2fcc51ff3939c7ae657d0726681ca1
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


There are several spelling mistakes in various ksmbd_err and
ksmbd_debug messages. Fix these.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/cifsd/ndr.c            | 2 +-
 fs/cifsd/smb2pdu.c        | 4 ++--
 fs/cifsd/transport_rdma.c | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/fs/cifsd/ndr.c b/fs/cifsd/ndr.c
index d96dcd9e43c6..aa0cb8fc555d 100644
--- a/fs/cifsd/ndr.c
+++ b/fs/cifsd/ndr.c
@@ -325,7 +325,7 @@ int ndr_decode_v4_ntacl(struct ndr *n, struct xattr_ntacl *acl)
 
 	ndr_read_bytes(n, acl->desc, 10);
 	if (strncmp(acl->desc, "posix_acl", 9)) {
-		ksmbd_err("Invalid acl desciption : %s\n", acl->desc);
+		ksmbd_err("Invalid acl description : %s\n", acl->desc);
 		return -EINVAL;
 	}
 
diff --git a/fs/cifsd/smb2pdu.c b/fs/cifsd/smb2pdu.c
index a4e78ebb0773..730bddbc8152 100644
--- a/fs/cifsd/smb2pdu.c
+++ b/fs/cifsd/smb2pdu.c
@@ -2508,7 +2508,7 @@ int smb2_open(struct ksmbd_work *work)
 	if (req->NameLength) {
 		if ((req->CreateOptions & FILE_DIRECTORY_FILE_LE) &&
 			*(char *)req->Buffer == '\\') {
-			ksmbd_err("not allow directory name included leadning slash\n");
+			ksmbd_err("not allow directory name included leading slash\n");
 			rc = -EINVAL;
 			goto err_out1;
 		}
@@ -2636,7 +2636,7 @@ int smb2_open(struct ksmbd_work *work)
 	}
 
 	if (!(req->DesiredAccess & DESIRED_ACCESS_MASK)) {
-		ksmbd_err("Invalid disired access : 0x%x\n",
+		ksmbd_err("Invalid desired access : 0x%x\n",
 			le32_to_cpu(req->DesiredAccess));
 		rc = -EACCES;
 		goto err_out1;
diff --git a/fs/cifsd/transport_rdma.c b/fs/cifsd/transport_rdma.c
index 1698f7ed9c2f..4f4806d67ab0 100644
--- a/fs/cifsd/transport_rdma.c
+++ b/fs/cifsd/transport_rdma.c
@@ -485,7 +485,7 @@ static int smb_direct_check_recvmsg(struct smb_direct_recvmsg *recvmsg)
 		struct smb2_hdr *hdr = (struct smb2_hdr *) (recvmsg->packet
 				+ le32_to_cpu(req->data_offset) - 4);
 		ksmbd_debug(RDMA,
-				"CreditGranted: %u, CreditRequested: %u, DataLength: %u, RemaingDataLength: %u, SMB: %x, Command: %u\n",
+				"CreditGranted: %u, CreditRequested: %u, DataLength: %u, RemainingDataLength: %u, SMB: %x, Command: %u\n",
 				le16_to_cpu(req->credits_granted),
 				le16_to_cpu(req->credits_requested),
 				req->data_length, req->remaining_data_length,
-- 
2.33.0

